torch/csrc/jit/passes/vulkan_rewrite.cpp

```diff
@@ -28,23 +28,13 @@ void insertPrePackedLinearOp(std::shared_ptr<Graph>& graph) {
   // fuse decomposed linear into aten::linear
   FuseLinear(graph);
 
-  std::string linear_before_inline = R"(
+  const std::string linear_before_inline = R"(
     graph(%linear, %input, %weight, %bias):
-        %r = prim::CallFunction(%linear, %input, %weight, %bias)
-        return (%r))";
-  std::string prepacked_ops_pattern_before_inline = R"(
-    graph(%linear, %input, %weight, %bias):
-        %weight_t = aten::t(%weight)
-        %packed_weight_bias = vulkan_prepack::linear_prepack(
-            %weight_t, %bias)
-        %res = vulkan_prepack::linear_run(%input, %packed_weight_bias)
+        %res = prim::CallFunction(%linear, %input, %weight, %bias)
         return (%res))";
-  std::string linear_pattern = R"(
-    graph(%input, %weight, %bias):
-        %r = aten::linear(%input, %weight, %bias)
-        return (%r))";
-  std::string prepacked_ops_pattern = R"(
-    graph(%input, %weight, %bias):
+
+  const std::string prepacked_ops_pattern_before_inline = R"(
+    graph(%linear, %input, %weight, %bias):
         %weight_t = aten::t(%weight)
```

Comment:
@SS-JIA why do you need to transport weight? I understand it will be const propagated but still.

Reply:
I recall it's because the prepacking function I wrote expects the weight to be transpose of how it's actually stored. It was easier to transpose the weight at this stage than to re-write the pre-packing logic.